src/silo/config/initialize_config.cpp

```diff
@@ -128,8 +141,8 @@ void InitializeConfig::overwriteFrom(const VerifiedConfigAttributes& config_sour
    if (auto var = config_source.getPath(inputDirectoryOptionKey())) {
       initialization_files.directory = var.value();
    }
-   if (auto var = config_source.getPath(lineageDefinitionsFilenameOptionKey())) {
-      initialization_files.lineage_definitions_file = var.value();
+   if (auto var = config_source.getString(lineageDefinitionFilenamesOptionKey())) {
+      initialization_files.lineage_definition_files = var.value();
```

Comment:
Do we maybe need config values that allow arrays as their value? From CLI those could be repeated arguments. In env variables we can still fall back to comma separated strings and parse them here. Anyway I think it would be good to do the parsing as early as possible (i.e. here and not only when reading `initialization_files.lineage_definition_files`).